sdk/turing/router/config/router_ensembler_config.py

```diff
@@ -23,6 +23,33 @@ def final_response_route_id(self, final_response_route_id: str):
     def to_open_api(self) -> OpenApiModel:
         return None
 
+@dataclass
+class EnsemblerStandardConfig:
```

Comment:
Maybe I'm not catching the motivation here, but what's the reason behind creating intermediary classes like `EnsemblerStandardConfig` and `EnsemblerNopConfig`?

Reply:
I'm happy you asked this. It's a bit of a question I was having myself - whether these classes were really needed. I have come to the conclusion we can do away with them, with some refactoring.

Why they exist today is, when the router config is parsed out of the API response (such as Get Router), we initialise a `RouterEnsemblerConfig` object as opposed to the child class objects. With the base class object, you need to rely on the `.nop_config` or `.standard_config` to get all the properties, which is what these intermediate classes achieve (the autogenerated Turing classes do not have the `final_response_route_id` or `fallback_response_route_id` properties and these are initialised on the intermediate data classes).

Ideally, we can init the proper child classes using some creator factory in which case the child classes alone will suffice to hold these new properties as well as encompass the autogenerated Turing classes for the standard config, pyfunc config, etc. I haven't dug into this yet but will either include it in my next PR which closes the loop on this feature or add it to the backlog.

Does it make sense?

Reply:
Yeah that makes perfect sense; the thing about the router config needing to be parsed out of the API response and having to handle the `final_response_route_id` or `fallback_response_route_id` properties completely slipped my mind.

>Ideally, we can init the proper child classes using some creator factory in which case the child classes alone will suffice to hold these new properties as well as encompass the autogenerated Turing classes for the standard config, pyfunc config, etc.

Yeah this would be perfect since it'll remove the potentially confusing naming of all the ensembler config classes (that are as of present only distinguishable from each other by the order in which words are being stuck together 😅); I kinda left the SDK in an awkward spot whereby the users get to create ensembler configs using the child classes when building a router config from scratch, but are forced to only manage the parent `RouterEnsemblerConfig` object when the router config conversely parsed out of the API response. But yeah this approach will definitely take a longer time to design/implement so it's definitely out of the scope of this PR. 

Thanks a lot for the current implementation and for the clarification!